saos-search/src/main/java/pl/edu/icm/saos/search/search/model/JudgeResult.java

```diff
@@ -33,6 +35,15 @@ public String getName() {
     }
 
     
+    //------------------------ LOGIC --------------------------
+    
+    public boolean isPresidingJudge() {
+    	return specialRoles.stream()
+				.filter(p -> p.equals(Judge.JudgeRole.PRESIDING_JUDGE))
+				.collect(Collectors.toList())
+				.size() > 0 ? true: false; 
+    }
+    
```

Comment:
Why not to use simply specialRoles.contains(JudgeRole.PRESIDING_JUDGE).
Also I would change this method to `hasRole(JudgeRole)`.